themeparks/disneyland.html

```diff
@@ -0,0 +1,323 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="disneyland.css" />
+    <script src="/fairyDustCursor.js"></script>
+    <title>Destination California</title>
+  </head>
+  <body>
+    <img id="header" src="/photos/losangeles5.jpeg" alt="Disneyland" />

```

Comment:
Folosește adrese relative in loc de `/...`
```suggestion
    <img id="header" src="../photos/losangeles5.jpeg" alt="Disneyland" />
```